### PR TITLE
Copter: add Copter::option_enabled(...) replacing bitops vs copter.g2…

### PR DESCRIPTION
**Pull request by peterbarker**
….flight_options

I've already added a test for an option bit in autotest and played with same in SITL

### CODE REVIEW

File: ArduCopter/Copter.h [hunk 1]
```diff
@@ -622,11 +622,13 @@ class Copter : public AP_Vehicle {
     };
 
 
-    enum class FlightOptions {
+    enum class FlightOption {
```
Comment (peterbarker) [inline ArduCopter/Copter.h:625]:
```suggestion
    enum class FlightOption : uint32_t {
```

**Comment by peterbarker** [comment]
Now also fixes intermittent new test by fixing the Gripper code to emit messages correctly.

File: libraries/AP_Gripper/AP_Gripper_Servo.cpp [hunk 2]
```diff
@@ -104,8 +104,10 @@ void AP_Gripper_Servo::update_gripper()
     // Check for successful grabbed or released
     if (config.state == AP_Gripper::STATE_GRABBING && has_state_pwm(config.grab_pwm)) {
         config.state = AP_Gripper::STATE_GRABBED;
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Gripper load grabbed");
```
Comment (rmackay9) [inline libraries/AP_Gripper/AP_Gripper_Servo.cpp:107]: Does this perhaps result in duplicate messages being output?  I see we're already outputting these messages higher up in the file

**Comment by peterbarker** [comment]
On Wed, 19 Jun 2024, Randy Mackay wrote:

> Does this perhaps result in duplicate messages being output? I see we're already outputting these messages higher up in the file

Only if "release" is called again - and in that case we're supposed to let 
the user know that we've done that, so duplkicates are in the design.

**Review by lthall — APPROVED** [review]

**Pull request merged**